src/openforms/forms/tests/test_api_formdefinition.py

```diff
@@ -156,6 +156,35 @@ def test_update(self):
         self.assertEqual(True, definition.login_required)
         self.assertIn({"label": "New field"}, definition.configuration["components"])
 
+    def test_update_is_reusable_unsuccessful_with_multiple_forms(self):
+        user = StaffUserFactory.create(user_permissions=["change_form"])
+        self.client.force_authenticate(user=user)
+
+        step = FormStepFactory.create(
+            form__internal_name="InternalForm",
+            form_definition__internal_name="InternalDef",
+            order=1,
+        )
+        FormStepFactory.create(
+            form_definition=step.form_definition,
+        )
+
+        url = reverse(
+            "api:formdefinition-detail", kwargs={"uuid": step.form_definition.uuid}
+        )
+        response = self.client.patch(
+            url,
+            data={
+                "isReusable": False,
+            },
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(
+            response.json()["invalidParams"][0]["reason"],
+            "Cannot change is_reusable from true to false if the form definition is used in more than one formstep.",
```

Comment:
**Mistake**

Assertion message is not marked translatable. This test will crash when we update the Dutch translations.